Classes/Emogrifier.php

```diff
@@ -298,7 +306,17 @@ public function emogrify()
 
         $this->copyCssWithMediaToStyleNode($cssParts, $xmlDocument);
 
-        return $xmlDocument->saveHTML();
+        if ($this->returnBodyContentOnly) {
```

Comment:
Yes, I love that approach.

I see two possible approaches (which are both completely fine with me):

a) create a separate PR for the refactoring and then continue with this PR here after I've merged the refactoring PR
b) do the refactoring as part of this PR